plugins/extractit/icon_extractit/util/extractor.py

```diff
@@ -12,15 +12,22 @@
 from pdfminer.pdfparser import PDFSyntaxError
 
 
-def extract(provided_regex: str, provided_string: str, provided_file: str) -> list:
+def extract(provided_regex: str, provided_string: str, provided_file: str, keep_original_url=False) -> list:
```

Comment:
Is it ok to set the parameter default for keep_original_url here?

Reply:
That should be fine.